Gordon360/Services/RecIM/ActivityService.cs

```diff
@@ -140,10 +140,29 @@ public async Task<ActivityCreatedViewModel> PostActivity(ActivityUploadViewModel
             };
             await _context.Activity.AddAsync(activity);
             await _context.SaveChangesAsync();
+
             return activity;
         }
 
-    }
+        public async Task<ParticipantActivityCreatedViewModel> PostParticipantActivityAsync(string username, int activityID, int privTypeID, bool isFreeAgent)
+        {
+            var participantActivity = new ParticipantActivity
+            {
+                ActivityID = activityID,
+                ParticipantUsername = username,
+                PrivTypeID = privTypeID,
+                IsFreeAgent = isFreeAgent,
+            };
+            await _context.ParticipantActivity.AddAsync(participantActivity);
+            await _context.SaveChangesAsync();
 
+            return participantActivity;
+        }
+
+        public bool IsReferee(string username, int activityID)
+        {
+            return _context.ParticipantActivity.Any(pa => pa.ParticipantUsername == username && pa.ActivityID == activityID);
```

Comment:
@Kevin-Chu58, this is a good point. 

@cpabbot our current workflow DOES only add people into ParticipantActivity if they have some level of priv. However, this is probably a mistake and should account for all participants of an activity.

Reply:
@amos-cha So all participants in an activity should be referees of that activity?

Reply:
I can't confirm, but the DB should always be updated with the Lucid graph
@cpabbot the reason I argue that there should be instances of every participant for an activity in ParticipantActivity is so that we can give Admins the ability to see all people in an Activity as well as being able to http PATCH permissions rather than adding and deleting instances